job_executor/worker/build_metadata_worker.py

```diff
@@ -13,6 +13,16 @@
     dataset_transformer
 )
 
+WORKING_DIR = local_storage.WORKING_DIR
+
+
+def _clean_working_dir(dataset_name: str):
+    generated_files = [
+        WORKING_DIR / f'{dataset_name}.json'
+    ]
+    for file_path in generated_files:
+        local_storage.delete_working_dir_file(file_path)
```

Comment:
This raises a LocalStorageError if file_path is not in the working directory (= the file does not exist there), whereas what we want here is to not fail if any of the files does not exist since we don't know at which step the process failed.

Reply:
That's true, my bad!